src/features/game/components/bank/components/Withdraw.tsx

```diff
@@ -147,47 +167,75 @@ export const Withdraw: React.FC<Props> = ({ onClose }) => {
+    return (
       <div className="p-2 flex flex-col justify-center space-y-1">
-        <span className="text-sm mb-1">{t("withdraw.sync")}</span>
+        <span className="mb-1">{t("withdraw.sync")}</span>
         <div className="flex space-x-1">
           <Button onClick={() => setPage("tokens")}>
-            <div className="flex">
-              <img src={token} className="h-4 mr-1" />
-              {"SFL"}
+            <div className="flex items-center">
+              <img src={getPageIcon("tokens")} className="h-4 mr-1" />
+              {getPageText("tokens")}
             </div>
           </Button>
           <Button onClick={() => setPage("resources")}>
-            <div className="flex">
-              <img src={SUNNYSIDE.resource.wood} className="h-4 mr-1" />
-              {t("resources")}
+            <div className="flex items-center">
+              <img src={getPageIcon("resources")} className="h-4 mr-1" />
+              {getPageText("resources")}
             </div>
           </Button>
         </div>
         <div className="flex space-x-1">
           <Button onClick={() => setPage("items")}>
-            <div className="flex">
-              <img src={chest} className="h-4 mr-1" />
-              {t("collectibles")}
+            <div className="flex items-center">
+              <img src={getPageIcon("items")} className="h-4 mr-1" />
+              {getPageText("items")}
             </div>
           </Button>
           <Button onClick={() => setPage("wearables")}>
-            <div className="flex">
-              <img src={SUNNYSIDE.icons.wardrobe} className="h-4 mr-1" />
-              {t("wearables")}
+            <div className="flex items-center">
+              <img src={getPageIcon("wearables")} className="h-4 mr-1" />
+              {getPageText("wearables")}
             </div>
           </Button>
         </div>
         <div className="flex space-x-1">
           <Button onClick={() => setPage("buds")}>
-            <div className="flex">
-              <img src={SUNNYSIDE.icons.plant} className="h-4 mr-1" />
-              {t("buds")}
+            <div className="flex items-center">
+              <img src={getPageIcon("buds")} className="h-4 mr-1" />
+              {getPageText("buds")}
             </div>
           </Button>
         </div>
       </div>
+    );
+  };
+
+  const navigationMenu = () => {
+    return (
+      <div className="flex items-center">
+        <img
+          src={SUNNYSIDE.icons.arrow_left}
+          className="self-start cursor-pointer mr-3"
+          style={{
+            top: `${PIXEL_SCALE * 2}px`,
+            left: `${PIXEL_SCALE * 2}px`,
+            width: `${PIXEL_SCALE * 11}px`,
+          }}
+          alt="back"
+          onClick={() => setPage("main")}
+        />
+        <Label type="default" icon={getPageIcon(page)}>
+          {getPageText(page)}
+        </Label>
+      </div>
+    );
+  };
+
+  return (
+    <>
+      {page === "main" && mainMenu()}
```

Comment:
In this line you can either do 
```{page === "main" && (<the entire component>)}```
OR put the mainMenu Component in a separate React Component then declare it here